src/libs/ActiveClientManager/index.js

```diff
@@ -36,8 +49,9 @@ Onyx.connect({
  * Add our client ID to the list of active IDs
  */
 function init() {
-    ActiveClients.addClient(clientID)
-        .then(resolveIsReadyPromise);
+    activeClients = _.without(activeClients, clientID);
+    activeClients.push(clientID);
```

Comment:
I believe the intention of doing it like this is to ensure that the clientID is always added at the end of the array and its only added once. Would you mind updating the method description to talk about that? I know it's mentioned pretty well in the comment at the top of the file, but having the method doc comment talk about it is a little better IMO since it's closer to the code.

Reply:
I actually don't get why are we concerned with it only being added once. 

We only add more clients and take them away to keep the list of IDs from growing infinitely. The active client will always be the last one regardless of whether we clean the current user clientID.

Reply:
I agree with Marc, but it's a NAB for me on this PR. Maybe a follow-up